src/store/plugins/confirmations.js

```diff
@@ -0,0 +1,33 @@
+import Vue from 'vue';
+import { i18n } from '../../popup/utils/i18nHelper';
+
+const confirmationsText = {
+  login: i18n.t('pages.confirmations.login'),
+  messageSign: i18n.t('pages.confirmations.message-sign'),
+  transactionSign: i18n.t('pages.confirmations.transaction-sign'),
+};
+
+export default store =>
+  store.registerModule('confirmations', {
+    namespaced: true,
+    state: {
+      undeniable: {
+        login: false,
```

Comment:
I would name it `address` because proper logging in requires some proof like message signing.